### PR TITLE
LRQA-53654

### PR DESCRIPTION
**Pull request by Adriverforlife**

### CODE REVIEW

**Comment by Adriverforlife** [comment]
"ci:test

**Comment by Adriverforlife** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase [hunk 1]
```diff
@@ -817,7 +817,7 @@ definition {
 
 		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
 
-		WorkflowMetrics.waitForSLAtoBecomeOverdue(timesToRun = "12");
+		WorkflowMetrics.waitForSLAtoBecomeOverdue(timesToRun = "12", key_expectedValue = "0");
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: Hi @Adriverforlife ,
Is the key_expectedvalue really 0? Or should it be 1?

**Comment by barrosdennys** [comment]
ci:test:workflow

File: portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase (hunk 1 above)
Comment (Adriverforlife) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: expected value is the expected value for it to continue to run. I can name it something else if you would like. Or I could do expectedValue -1

Comment (barrosdennys) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: Got it!! No, no need to change it! Thank you very much!

**Comment by barrosdennys** [comment]
PR sent to BChan:
https://github.com/brianchandotcom/liferay-portal/pull/84075

**Pull request closed without merging**